_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,172 @@
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any `element` that is [visible](#visible) and has a CSS `transform` property with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) transform-function, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`landscape` or `portrait`).
+
+## Expectation 1
+
+Each target element is not restricted to a single display orientation using CSS rotate transform-function, whose value is a multiple of 90 degrees but not a multiple of 180 degrees.
+
+## Assumptions
+
+- The rule does not consider the existence of any control on the page that can change the orientation on demand.
+
+## Accessibility Support
+
+*There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- [Understanding Success Criterion 1.3.4: Orientation](https://www.w3.org/WAI/WCAG21/Understanding/orientation.html)
+- [Managing screen orientation](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model/Managing_screen_orientation)
+- [MDN Orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+A page, where CSS media query styles negate orientation lock, and therefore does not restrict its view or operation.
+
+```html
+<html lang="en">
+  <head>
+    <!--
+      1. The style from the relative stylesheet applies `transform: rotate(90deg)`
+    -->
+    <link rel="stylesheet" href="../test-assets/css-orientation-lock-media-queries-b33eff/html-css-lock.css">
+    <!--
+      2. The below style negates the orientation lock
+    -->
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(-90deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <main>
+      Page Content
+    </main>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+A page, where orientation is locked by applying `transform` property on `portrait` media query (coming from a relative stylesheet).
+
+```html
+<html lang="en">
+  <head>
+    <!--
+      1. The style from the relative stylesheet applies `transform: rotate(90deg)`
+    -->
+    <link rel="stylesheet" href="../test-assets/css-orientation-lock-media-queries-b33eff/html-css-lock.css">
+  </head>
+  <body>
+      Page Content
+  </body>
+</html>
+```
+
+#### Failed Example 2
+
+A page, where orientation is locked via `style` specified on the `head`.
+
+```html
+<html lang="en">
+  <head>
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(-90deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+      Page Content
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+A page where there are no styles.
+
+```html
+<html lang="en">
+  <body>
+    I am a page with no styles
+  </body>
+</html>
+```
+
+#### Inapplicable Example 2
```

Comment:
For the purposes of this rule, there is no difference between this and the first inapplicable example

Reply:
I have modified this slightly. Good find though.